llpc/util/llpcUtil.h

```diff
@@ -129,6 +130,28 @@ const char *getUnlinkedShaderStageName(Vkgc::UnlinkedShaderStage type);
 inline bool doesShaderStageExist(llvm::ArrayRef<const PipelineShaderInfo *> shaderInfo, ShaderStage stage) {
   return stage < shaderInfo.size() && shaderInfo[stage] && shaderInfo[stage]->pModuleData;
 }
+
+// Returns true iff `stage` is present in the `stageMask`.
```

Comment:
Please add separate line "// ====" for each function according to the coding style even if they are defined in header file.

Reply:
Done